doc/import.rst

```diff
@@ -0,0 +1,81 @@
+Importing SpikeInterface
+========================
+
+SpikeInterface allows for the generation of powerful and reproducible spike sorting pipelines.
+Flexibility is built into the package starting from import to maximize the productivity of
+the developer and the scientist. Thus there are three ways that SpikeInterface and its components
+can be imported:
+
+
+Importing by Module
+-------------------
+
+Since each spike sorting pipeline involves a series of often repeated steps, many of the developers
+working on SpikeInterface recommend importing in a module by module fashion. This will allow you to
+keep track of your processing steps (preprocessing, postprocessing, quality metrics, etc.). This can
+be accomplished by:
+
+.. code-block:: python
+
+    import spikeinterface as si
+
+to import the :code:`core` module followed by:
+
+.. code-block:: python
+
+    import spikeinterface.extractors as se
+    import spikeinterface.preprocessing as spre
+    import spikeinterface.sorters as ss
+    import spikinterface.postprocessing as spost
+    import spikeinterface.qualitymetrics as sqm
+    import spikeinterface.exporters as sexp
+    import spikeinterface.comparsion as scmp
+    import spikeinterface.curation as scur
+    import spikeinterface.sortingcomponents as sc
+    import spikeinterface.widgets as sw
+
+to import any of the other modules you wish to use.
+
+The benefit of this approach is that it is lighter and faster than importing the whole package and allows
```

Comment:
I am not sure this is true. The picture is actually quite muddy because of our use of try-except in many parts of the library. Check out the import time profiling in this PR:

https://github.com/SpikeInterface/spikeinterface/actions/runs/6798751083

Reply:
So I think I will stress the mnemonic effects that you were arguing for in the issue. We really don't have performance guarantees yet (but I would love to add that in the future).

Reply:
All right, then the same as below. Feel free to ignore it and I can correct it later. 
I wanted to make you aware that there are actually not great performance differents in import time and show you that we keep a profile of that in the unittests. 

As we become better that difference should increase but it is not very large right now as you can see in the link.

Reply:
I think this one I'll change. Until we get everything optimized there's no point in being misleading (ie the chance the documentation gets changed is lower just because it drops off people's todo list). Looks like extractors and comparison are the big offenders. Too bad extractors are part of each workflow.